### ISSUE TITLE
remove the option to return c_nonce from the credential response

### ISSUE THREAD

**Issue by bc-pi** [openid/OpenID4VCI#393]
This only makes sense when/if/after #381 is merged. 

_Originally posted by @bc-pi in relation to https://github.com/openid/OpenID4VCI/issues/381#issuecomment-2357047069_

> > Discussed on the WG call, direction on removing section 7.3.2 and consolidating its content in Nonce Endpoint definition. https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html#section-7.3.2

> While I was part of that discussion, after looking again at that section and content of this PR, I don't see how that consolation could actually manifest in a meaningful way. I suggest this PR proceed without and consider that as part of the soon to be separate issue on removing an option to return c_nonce from the credential [error] response.

 > > also agreed to open a separate issue on removing an option to return c_nonce from the credential error response, so that we can merge this PR as-is (@bc-pi).

that is this

**Comment by bc-pi** [comment]
I guess there's the potential to consider two different things here: 1) `c_nonce` in a credential *error* response and 2) `c_nonce` in a _successful_ credential response.

But I'm mostly of the mind to treat it as all or nothing.

**Comment by bc-pi** [comment]
when/if/after ... 

![Screenshot 2024-09-24 at 10 54 23 AM](https://github.com/user-attachments/assets/718a6e97-2eea-4ebb-bfb9-61fc004bf32b)
